_includes/samples/VRF/VRFv2SubscriptionManager.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+// Example of a single consumer contract which owns the subscription.
+pragma solidity ^0.8.7;
+
+import "@chainlink/contracts/src/v0.8/interfaces/LinkTokenInterface.sol";
+import "@chainlink/contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol";
+import "@chainlink/contracts/src/v0.8/dev/VRFConsumerBaseV2.sol";
+
+contract VRFv2Consumer is VRFConsumerBaseV2 {
```

Comment:
The name of this contract is the same as the previous file - perhaps this should be VRFv2SubscriptionManager instead?